WordPress/wordcamp.org
# Fix GitHub & SVN file syncing issue

To avoid confusion, here's the approach being taken:

1. Just like with SVN, revert the two commits that caused the issue.
2. Then, reapply the "Needs Follow-up" status that didn't cause any problems.


See https://wordpress.slack.com/archives/C08M59V3P/p1716304671009819.


Props @pkevan 



### How to test the changes in this Pull Request:

1. Log in as a network admin
2. Open one and edit it, the staus should be visible
3. Visit wp-admin/network/admin.php?page=reimbursement-requests-dashboard and tab "Needs follow-up" should be there
4. On a WordCamp site, create a vendor payment and set due date to past date, set the status to "Needs follow-up"
5. Visit /wp-admin/network/admin.php?page=wcp-dashboard and the payment should be listed on "Overdue" tab and separate "Needs follow-up" tab should be visible
6. Reimbursement page should work as usual
